packages/contracts-bedrock/deploy-config/devnetL1.json

```diff
@@ -1,47 +1,47 @@
 {
-  "l1ChainID": 900,
-  "l2ChainID": 901,
-  "l2BlockTime": 2,
+  "l1ChainID": 901,
+  "l2ChainID": 902,
+  "l2BlockTime": 1,
   "maxSequencerDrift": 300,
   "sequencerWindowSize": 200,
   "channelTimeout": 120,
   "p2pSequencerAddress": "0x9965507D1a55bcC2695C58ba16FB37d819B0A4dc",
-  "batchInboxAddress": "0xff00000000000000000000000000000000000000",
+  "batchInboxAddress": "0xff00000000000000000000000000000000000901",
   "batchSenderAddress": "0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC",
-  "l2OutputOracleSubmissionInterval": 20,
-  "l2OutputOracleStartingTimestamp": -1,
+  "cliqueSignerAddress": "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
+  "l1UseClique": true,
+  "l1StartingBlockTag": "earliest",
+  "l2OutputOracleSubmissionInterval": 6,
```

Comment:
see if we can increase this safely

Reply:
added adee57d17a7708a566108cc0d17ae7dd433f5ea2 to get logging on the rate of output proposals vs L2 blocks. If L2 is running away from L1 it makes the docker devnet setup unusable for dev. Will fix based on the results of the logging

Reply:
Can confirm that the devnet will not have issues with an interval of 6 with the current L1 and L2 blocktimes